localization/languages/nl.json

```diff
@@ -0,0 +1,251 @@
+        "settingsContentBlockingLevel0": "Alle advertenties en trackers toestaan",
+        "settingsContentBlockingLevel1": "Advertenties en trackers van derden blokkeren",
+        "settingsContentBlockingLevel2": "Alle advertenties en trackers blokkeren",
+        "settingsContentBlockingExceptions": "Advertenties worden toegestaan op volgende websites:",
+        "settingsBlockScriptsToggle": "Blokkeer scripts",
+        "settingsBlockImagesToggle": "Blokkeer afbeeldingen",
+        "settingsEasyListCredit": {
+            "unsafeHTML": "Gebaseerd op EasyList en EasyPrivacy, gecreëerd door <a target=\"_blank\" href=\"https://easylist.to/\">de EasyList auteurs</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>)."
+        },
+        "settingsAppearanceHeading": "Uitzicht",
+        "settingsEnableDarkMode": "Donkere modus inschakelen:",
+        "settingsDarkModeNever": "Nooit",
+        "settingsDarkModeNight": "'s Nachts",
+        "settingsDarkModeAlways": "Altijd",
+        "settingsSiteThemeToggle": "Kleurthema van site inschakelen",
+        "settingsAdditionalFeaturesHeading": "Aanvullende opties",
+        "settingsUserscriptsToggle": "Gebruikerscripts inschakelen",
+        "settingsSeparateTitlebarToggle": "Toon de titelbalk",
+		"settingsUserscriptsExplanation": {"unsafeHTML": "Gebruikerscripts maken het mogelijk het gedrag van websites aan te passen - <a href=\"https://github.com/minbrowser/min/wiki/userscripts\">meer informatie</a>."},
+        "settingsUserAgentToggle": "Aangepaste user agent gebruiken",
+        "settingsUpdateNotificationsToggle": "Automatisch controleren op updates",
+        "settingsSearchEngineHeading": "Zoekmachine",
+        "settingsDefaultSearchEngine": "Kies uw standaard zoekmachine:",
+        "settingsDDGExplanation": "Stel DuckDuckGo in als de standaard zoekmachine om onmiddellijk resultaten in de zoekbalk te zien.",
+        "customSearchEngineDescription": "Vervang de zoekterm met %s",
+        "settingsKeyboardShortcutsHeading": "Sneltoetsen",
+        "settingsKeyboardShortcutsHelp": "Gebruik komma's om verschillende sneltoetscombinaties te scheiden.",
+        /* app menu */
+        "appMenuFile": "Bestand",
+        "appMenuNewTab": "Nieuw Tabblad",
+        "appMenuDuplicateTab": "Tabblad dupliceren",
+        "appMenuNewPrivateTab": "Nieuw Privé Tabblad",
+        "appMenuNewTask": "Nieuwe Taak",
+        "appMenuSavePageAs": "Pagina Opslaan Als",
+        "appMenuPrint": "Afdrukken",
+        "appMenuEdit": "Bewerken",
+        "appMenuUndo": "Ongedaan maken",
+        "appMenuRedo": "Opnieuw doen",
+        "appMenuCut": "Knippen",
+        "appMenuCopy": "Kopiëren",
+        "appMenuPaste": "Plakken",
+        "appMenuSelectAll": "Alles selecteren",
+        "appMenuFind": "Zoeken",
+        "appMenuView": "Beeld",
+        "appMenuZoomIn": "Inzoomen",
+        "appMenuZoomOut": "Uitzoomen",
+        "appMenuActualSize": "100%",
```

Comment:
Is there a way to keep this in words so that it's consistent with the other menu items? Safari uses "werkelijke grootte" and Chrome uses "normaal"; would one of those work?

Reply:
In that case I'd go with 'Werkelijke grootte', although in my opinion 100% will do just fine.